test/integration/test_api.py

```diff
@@ -614,6 +614,71 @@ def mock_get(*args, **kwargs):
     assert body['errors'][0]['desc'] == 'The borehole exporter returned an error.'
 
 
+def test_get_ags_exporter_by_polygon(client):
+    # Arrange
+    # There should be 4 boreholes within 2 projects in this area
+    polygon = 'POLYGON((-3.946 56.063,-3.640 56.063,-3.640 55.966,-3.946 55.966,-3.946 56.063))'
+    query = f'/ags_export_by_polygon/?polygon={polygon}'
+    # Define the borehole and project IDs and zipped AGS file to use for the test
+    bgs_loca_ids = ['20200205093727287903', '20200205093728297906', '20200205093728297908', '20200205093728297910']
+    bgs_proj_ids = {id_[:16] for id_ in bgs_loca_ids}
+    ags_file_names = {f'{id_}.ags' for id_ in bgs_proj_ids}
+    ags_metadata_file_name = 'FILE/BGSFileSet01/BGS_download_metadata.txt'
+
+    # Act
+    with client as ac:
+        response = ac.get(query)
+
+    # Assert
+    assert response.status_code == 200
+    assert response.headers["Content-Disposition"] == 'attachment; filename="boreholes.zip"'
+    assert response.headers["Content-Type"] == "application/x-zip-compressed"
+    assert len(response.content) > 0
+
+    assert zipfile.is_zipfile(BytesIO(response.content))
+    with zipfile.ZipFile(BytesIO(response.content)) as ags_zip:
```

Comment:
Here I am just testing that the correct files are in the zip and that the IDs appear in the text file. The test, `test_get_ags_export_single_id` drills down into that zip which is sufficient, I think.